### PR TITLE
Add page_count when using Paginators

### PR DESCRIPTION
**Pull request by phantomphildius**
@tiagopog i noticed @lucaskuan [pr](https://github.com/tiagopog/jsonapi-utils/pull/74) for [this issue](https://github.com/tiagopog/jsonapi-utils/issues/74) and decided to help out since i could use this in my own project.

Would it be possible to take a look and provide feedback and merging if you think its acceptable?

Thanks!

### CODE REVIEW

**Comment by tiagopog** [comment]
Thanks for the contribution, @phantomphildius. I will take a look on this patch later today. Cheers!

File: lib/jsonapi/utils/support/pagination.rb [hunk 1]
```diff
@@ -157,6 +157,23 @@ def count_records(records, options)
...
+        #   e.g.: 42
+        #
+        # @return [Integer]
+        #   e.g 5
+        #
+        # @api private
+        def page_count_for(record_count)
+          return 0 if record_count.to_i < 1
+
+          size = page_params['size'] || page_params['limit']
```
Comment (tiagopog) [inline lib/jsonapi/utils/support/pagination.rb]: What about to apply the type casting on `size` only once?

```ruby
size = (page_params['size'] || page_params['limit']).to_i
size = JSONAPI.configuration.default_page_size unless size.nonzero?
(record_count.to_f / size).ceil
```

**Comment by tiagopog** [comment]
Closes #74 #76

**Comment by phantomphildius** [comment]
@tiagopog will this be released shortly or should I just pin my gem to the master branch for now? Also, let me know if you need help on other things, I would be happy to assist here and there!

**Comment by tiagopog** [comment]
I've just bumped the gem's version to `v0.7.2` :-)

**Pull request merged**